dkpro-core-io-nyt-asl/src/main/resources/desc/type/NYTArticleMetaData.xml

```diff
@@ -0,0 +1,239 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<typeSystemDescription xmlns="http://uima.apache.org/resourceSpecifier">
+	<name>NYTArticleMetaData</name>
+	<description />
+	<version>1.0</version>
```

Comment:
The version should be automatically set through Maven.

Here set:

```
  <version>${version}</version>
```

In the POM you need to add:

```
	<build>
		<resources>
			<resource>
				<filtering>false</filtering>
				<directory>src/main/resources</directory>
				<excludes>
					<exclude>desc/type/**/*</exclude>
				</excludes>
			</resource>
			<resource>
				<filtering>true</filtering>
				<directory>src/main/resources</directory>
				<includes>
					<include>desc/type/**/*</include>
				</includes>
			</resource>
		</resources>
	</build>
```

And a file `src/filter/filter.properties` with the content

```
version=${project.version}
timestamp=${maven.build.timestamp}
```